core.py

```diff
@@ -1474,6 +1474,10 @@ def _setStroke(self, val):
 		self.element.setAttribute("stroke", val)
 
 
+class _isLeaf(object):
```

Comment:
We define our tag-name in the attribute `_baseClass`. I think it would be more consistent to have a boolean attribute to define the kind of an element.
Btw: Those elements called officially [void-elements](https://www.w3.org/TR/html5/syntax.html#void-elements). And you forget some of them.

Reply:
This is incorrect:
- the tag name in the HTML parser is determined by the class name
- the empty `_isLeaf()` class allows for checking on a class whether it is a sub-class of it or not WITHOUT, and thats the important point, instantiating the object behind the class. Latter one would result in huger runtime latency. Another solution would be write a class member variable after the class is defined, but this is ugly. The way I do it here works for PyJS, and hopefully we can do it using decorators somewhere in the future.

> Btw: Those elements called officially void-elements. And you forget some of them.

Void-element is a good naming, I will rename it. If I'd forgotten some tags you're welcome to mark them correctly.

Reply:
I've not pushed a new commit with renaming from _isLeaf to _isVoid and adding more elements. There are still some elements (e.g. base, col) which don't exist in html5 yet. But this is another topic.